### PR TITLE
Add support for Bionic

### PR DESCRIPTION
**Pull request by elmobp**
See the dep changes, also fixed tests to pin to xenial

### CODE REVIEW

File: spec/classes/rbenv_spec.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
-describe 'rbenv', type: 'class' do
-  let(:facts) { { osfamily: 'Debian' } }
+describe 'rbenv', :type => 'class' do
+  let(:facts) { { :osfamily => 'Debian', :lsbdistcodename => 'xenial' } }
```
Comment (jdowning) [inline spec/classes/rbenv_spec.rb]: You'll need to use the `{key: 'value'}` style hash for these changes.

File: manifests/deps/debian.pp [hunk 2]
```diff
@@ -3,16 +3,20 @@
...
     'build-essential',
     'git',
     'libreadline-dev',
-    'libssl-dev',
     'zlib1g-dev',
     'libffi-dev',
     'libyaml-dev',
     'libncurses5-dev',
     'libgdbm-dev',
+    $packages,
```
Comment (jdowning) [inline manifests/deps/debian.pp]: I'm not sure this works...inserting an array into an array? Perhaps this needs to use `merge` ?

Comment (elmobp) [reply manifests/deps/debian.pp]: merge could work to, we use it now

File: spec/classes/rbenv_spec.rb (hunk 1 above)
Comment (elmobp) [reply spec/classes/rbenv_spec.rb]: sorry i skipped over that by accident fixing now!

**Pull request merged**